cypress/e2e/pages/storyPage/tests.js

```diff
@@ -147,78 +148,63 @@ export const testsThatFollowSmokeTestConfig = ({
     mostReadAssertions({ service, variant });
   });
 
-  describe(`Recommendations on ${service} ${pageType}`, () => {
-    it('Recommendations have images', () => {
-      isArticleLessThanTwoYearsOld().then(runRecommendationTests => {
-        if (runRecommendationTests) {
-          cy.getToggles(service);
-          cy.fixture(`toggles/${service}.json`).then(toggles => {
-            const recommendationsEnabled = path(
-              ['cpsRecommendations', 'enabled'],
-              toggles,
+  // TODO: Remove once rolled out to all services
+  if (SERVICES_WITH_NEW_RECOMMENDATIONS.includes(service)) {
```

Comment:
Can this be a toggle instead of hardcoded config?

Reply:
Probably not worth it tbh. Its a very temporary measure until editorial get the go ahead from other services. Its to minimise the component unexpectedly appearing, so just some comms needing to go out first.